src/lib/components/workflow/batch-operation-confirmation-modal.svelte

```diff
@@ -97,17 +101,24 @@
           />
         </p>
       {/if}
+      <Input
+        id={`bulk-action-reason-${action}`}
+        bind:value={reason}
+        label={translate('reason')}
+        hintText={translate(
+          'workflows',
+          'batch-operation-confirmation-input-hint',
+          { placeholder },
+        )}
+        {placeholder}
+      />
+      <Input
+        id="batch-operation-job-id"
+        label={translate('job-id')}
+        hintText={translate('batch', 'job-id-input-hint')}
+        bind:value={jobId}
```

Comment:
TODO: validate this is URL-safe, as the jobID ends up in the URL

Reply:
Can we use the utils in `encode-uri.ts` for this? Maybe add something to `routeForBatchOperation` real quick?

Reply:
Gah sorry I missed this before merging. I suppose we could have done that but I think this is also a totally reasonable approach.